content/en/releases/download.md

```diff
@@ -70,6 +72,11 @@ those derivations are signed in the same way as the multi-architecture manifest
 
 <!-- TODO: describe how to use the signed images -->
 
+Full list of images that are signed can be found [here](/examples/admin/signed-images/auto-generated-list-of-all-signed-images.txt)
```

Comment:
Can we move the signature information into the Markdown table, perhaps?

Reply:
Given that the collection of core components is pretty stable at this stage, compiling this into a table seems doable.

Reply:
We are using SBOM to get list of images now. So resolving this comment